### PR TITLE
Updates to CC artifact list

### PR DESCRIPTION
**Pull request by peggygannon**
This PR includes updates to the following templates:
- staging
- design intent
- midpoint review
- Full accessibility audit

### CODE REVIEW

**Comment by peggygannon** [comment]
Hi, @shiragoodman Here is PR for updates to templates/artifacts list , for your review.

**Review by allison0034 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/design-intent-collaboration.md [hunk 1]
```diff
@@ -7,51 +7,90 @@ assignees: shiragoodman, allison0034
...
+### VFS Attendees
+<!-- Delete any of the below if they are not applicable for your meeting -->
+- Content and IA: 
+- Accessibility: 
+- QA specialists: 
+
 
 ## Scheduling
 
-### Design Intents can only be scheduled on Monday, Tuesday, or Friday.
```
Comment (shiragoodman) [inline .github/ISSUE_TEMPLATE/design-intent-collaboration.md:41]: did we remove "Design intents can only be scheduled on monday , tuesday and Friday." I would like to see this included as a comment.

Comment (peggygannon) [reply .github/ISSUE_TEMPLATE/design-intent-collaboration.md:41]: Hi, @shiragoodman it's still in there.. it's part of the content note that shows conditionally with the other scheduling bullet points.

File: .github/ISSUE_TEMPLATE/midpoint-review.md [hunk 2]
```diff
@@ -7,57 +7,124 @@ assignees: shiragoodman
...
 
 ## Scheduling
-Would you like this review to be asynchronous? Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review.
+Would you like this review to be asynchronous? 
+
+Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review.
 - [ ] Yes
 - [ ] No
- 
-### Midpoint Reviews can only be scheduled on Monday, Tuesday, or Friday.
```
Comment (shiragoodman) [inline .github/ISSUE_TEMPLATE/midpoint-review.md:41]: again, can we make sure this is included in the comments?

Comment (shiragoodman) [reply .github/ISSUE_TEMPLATE/midpoint-review.md:41]: nvm i see it below!

**Pull request merged**